src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+    let sum = 0;
+
+  function adder(num) {
```

Comment:
Consider naming the inner function something more descriptive than `adder`. While it does describe what the function does, a name like `addOrGetSum` might more clearly convey the dual functionality of adding numbers or returning the sum when no arguments are provided.